wheel/chia_rs.pyi

```diff
@@ -75,7 +75,7 @@ class AugSchemeMPL:
     @staticmethod
     def derive_child_sk_unhardened(pk: PrivateKey, index: int) -> PrivateKey: ...
     @staticmethod
-    def derive_child_pk_unhardened(pk: PublicKey, index: int) -> PublicKey: ...
```

Comment:
the `PublicKey` and `Signature` types are exported as `G1Element` and `G2Element` to python, to stay compatible with `blspy`